css/styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+	background: #0D1326;
+	color: white;
+}
+
+ nav {
+    margin: 0;
+    padding: 1em;
+    background: #0D1326 /*#856D00*/;
+    text-align: center;
+    font-family: 'Tangerine', serif;
+    font-style: caps;
+    font-size: 30px;
+    letter-spacing: 1.25px;
+}
+ article {
+    margin-left: 170px;
+    border-left: 1px solid gray;
+    padding: 1em;
+    overflow: hidden;
+    font-family: sans-serif;
+}
+h1 {
+	font-size: 80px;
+}
+
+h3 {
+	text-align: center;
+}
+
+.jkimage {
+	clear:left;
+	float: left;
+	width:300px; height:300px;
```

Comment:
you should move height to the next line for better readability and it follows convention.